### PR TITLE
NO-ISSUE: update ib-cli documentation

### PR DESCRIPTION
**Pull request by tsorya**
# Background / Context

# Issue / Requirement / Reason for change

# Solution / Feature Overview

# Implementation Details

# Other Information

### CODE REVIEW

File: ib-cli/README.md [hunk 1]
```diff
@@ -114,10 +105,65 @@ INFO[2024-02-12 15:59:24] installation ISO created at: ibi-iso-workdir/rhcos-ibi
...
+### Proxy Configuration
+
+```yaml
+proxy:
+  httpProxy: "http://proxy.example.com:8080"
+  httpsProxy: "http://proxy.example.com:8080"
+  noProxy: "no_proxy.example.com"
+```
+
+###  Mirror Registry
```
Comment (leo8a) [inline ib-cli/README.md:123]:
```suggestion
### Mirror Registry
```

File: ib-cli/README.md [hunk 2]
```diff
@@ -114,10 +105,65 @@ INFO[2024-02-12 15:59:24] installation ISO created at: ibi-iso-workdir/rhcos-ibi
...
+        - 192.168.126.1
+  routes:
+    config:
+      - destination: 0.0.0.0/0
+        metric: 150
+        next-hop-address: 192.168.126.1
+        next-hop-interface: enp1s0
+```
+
+### Additional flags
```
Comment (leo8a) [inline ib-cli/README.md:167]:
```suggestion
### Additional flags
```

File: ib-cli/README.md [hunk 3]
```diff
@@ -114,10 +105,65 @@ INFO[2024-02-12 15:59:24] installation ISO created at: ibi-iso-workdir/rhcos-ibi
...
+    config:
+      - destination: 0.0.0.0/0
+        metric: 150
+        next-hop-address: 192.168.126.1
+        next-hop-interface: enp1s0
+```
+
+### Additional flags 
+
+Can be found at https://github.com/openshift-kni/lifecycle-agent/blob/main/api/ibiconfig/ibiconfig.go#L17
```
Comment (leo8a) [inline ib-cli/README.md:169]:
```suggestion
Can be found at the ImageBasedInstallConfig struct (link to [Go code](https://github.com/openshift-kni/lifecycle-agent/blob/main/api/ibiconfig/ibiconfig.go#L17)).

```

File: ib-cli/README.md [hunk 4]
```diff
@@ -70,35 +70,26 @@ podman run -it -v `pwd`:/data --entrypoint cp $(LCA_IMAGE) /usr/local/bin/ib-cli
...
-  --extra-partition-start ${EXTRA_PARTITION_START} \
-  --lca-image ${LCA_IMAGE} \
-  --seed-image ${SEED_IMAGE} \
-  --seed-version ${SEED_VERSION} \
-  --auth-file ${AUTH_FILE} \
-  --pullsecret-file ${PS_FILE} \
-  --ssh-public-key-file ${SSH_PUBLIC_KEY} \
-  --dir ./${WORKDIR}
+
+ib-cli create-iso --dir <your workinr>
```
Comment (achuzhoy) [inline ib-cli/README.md:92]:
```suggestion
ib-cli create-iso --dir <your working directory>
```

**Review by eranco74 — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**